scripts/zones/Lower_Delkfutts_Tower/npcs/_544.lua

```diff
@@ -4,44 +4,35 @@
 -- Notes: Door opens when you trade Delkfutt Key to it
 -- !pos 345 0.1 20 184
 -----------------------------------
-require("scripts/globals/settings");
-require("scripts/globals/keyitems");
-local ID = require("scripts/zones/Lower_Delkfutts_Tower/IDs");
+local ID = require("scripts/zones/Lower_Delkfutts_Tower/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
 -----------------------------------
 
-function onTrade(player,npc,trade)
-
-    if (trade:hasItemQty(549,1) and trade:getItemCount() == 1) then -- Trade Delkfutt Key
-        player:startEvent(16);
+function onTrade(player, npc, trade)
+    if npcUtil.tradeHas(trade, 549) then -- Delkfutt Key
+        player:startEvent(16)
     end
+end
 
-end;
-
-function onTrigger(player,npc)
-
-    if (player:hasKeyItem(dsp.ki.DELKFUTT_KEY)) then
-        player:startEvent(16);
+function onTrigger(player, npc)
+    if player:hasKeyItem(dsp.ki.DELKFUTT_KEY) then
+        player:startEvent(16)
     else
-        player:startEvent(10); -- door is firmly shut
+        player:startEvent(10) -- door is firmly shut
     end
 
-    return 1;
+    return 1
```

Comment:
while we're in lower delkfutt scripts would you mind nuking this return in each script and making sure that doesn't cause an issue? the onTrigger only has these 2 possibilities and both of them result in an event call, so the door -shouldn't- need the return to keep it from popping open on the client.

Reply:
if its a bother I can merge as is and come back to it later

Reply:
will check it this evening

Reply:
tested. the returns do nothing, whether they return 1 or 0. removed them all.